tools/cli/admin.go

```diff
@@ -139,6 +139,31 @@ func newAdminShardManagementCommands() []cli.Command {
 				AdminDescribeShard(c)
 			},
 		},
+		{
+			Name:    "list",
+			Aliases: []string{"l"},
+			Usage:   "List shard distribution",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagRole,
+					Value: "cadence-history",
+					Usage: "Role of the shard",
```

Comment:
I am little bit confused by the meaning of role here. Is `cadence-history` the only accepted value or we plan to add more like the tasklist distribution for matching?

Reply:
Value here is the default value, you can pass any value. I made it a parameter so we can gather other type of shards in the future if we wanted to.